### PR TITLE
Added favicons

### LINKED ISSUE #200: Add favicon
cuz it's cute :+1:

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #200

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <title>Mozfest 2015</title>
   <link rel="stylesheet" href="index.css"/>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png" />
```
Comment (Pomax) [inline public/index.html]: might as well add the 180x180 icon as normal icon:
```
<link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png">
<link rel="icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
```

Also note that `<link>` elements are void elements, and so don't need that `/>` at the end (just like ``, `<meta>`, `<br>`)

Comment (mmmavis) [inline public/index.html]: :+1: I should be more careful about this too

**Comment by mmmavis** [comment]
I did some cleanup to `<head>`. r? again plz @Pomax :smile_cat:

File: public/index.html [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
-  <link rel="stylesheet" type="text/css" href='https://css.tito.io/v1' />
-  <link rel="stylesheet" type="text/css" href='https://api.tiles.mapbox.com/mapbox.js/v2.1.5/mapbox.css' />
-  <link rel="stylesheet" href="build/bundle.css">
+  <title>Mozfest 2015</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png">
+  <link rel="icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:800,600,400,300">
+  <link rel="stylesheet" type="text/css" href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css">
+  <link rel="stylesheet" type="text/css" href="index.css">
```
Comment (mmmavis) [inline public/index.html]: ummm I'm not sure if loading `index.css` before `https://css.tito.io/v1`, `mapbox.css`, and `build/bundle.css` is intended. Let's just keep it as is for now

**Comment by Pomax** [comment]
r+!

**Pull request merged**